UI/web-src/ngMango/components/eventHandlerList/eventHandlerList.js

```diff
@@ -28,7 +28,12 @@ class EventHandlerListController {
         this.maEventHandler.subscribe({
             scope: this.$scope,
             handler: (event, item, attributes) => {
-                attributes.updateArray(this.eventHandlers, item => !this.eventType || item.hasEventType(this.eventType.typeId));
+                attributes.updateArray(this.eventHandlers, (eh) => {
+                    if (!this.eventType) return true;
+                    const found = eh || this.eventHandlers.find((i) => i.id === attributes.itemId);
+                    if (found) return found.hasEventType(this.eventType.typeId);
```

Comment:
![image](https://user-images.githubusercontent.com/17141727/137758920-bd7b1aa5-ea43-400b-97bd-8ebb067135ba.png)

When filter applied to the page, `item.hasEventType(this.eventType.typeId)` throws an error since item is null on delete.
Previously list was not updated with filters correctly.

I just implement workaround to get deleted item and decide either update array or not based on filter and deleted item event type

Reply:
Ok got it. Thanks.